### PR TITLE
Add test case for configuring r related feedstocks. Fix compute_build_matrix in combination with build->skip.

### PR DESCRIPTION
**Pull request by johanneskoester**
This test case is able to reproduce the problems discussed in PR #453 (see [here](https://github.com/conda-forge/conda-smithy/pull/453#issuecomment-280058833)).
Further, the PR also contains the resulting fix.

@ocefpaf @jakirkham please let me know what you think.

### CODE REVIEW

**Comment by johanneskoester** [comment]
Great, I was able to reproduce the error with the test case, see [here](https://travis-ci.org/conda-forge/conda-smithy/jobs/210861401#L443). Commit 75dc6bf contains a possible fix.

**Comment by ocefpaf** [comment]
Awesome @johanneskoester! Thanks for tackling this!!

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -603,6 +603,10 @@ def meta_of_feedstock(forge_dir, config=None):
 
 
 def compute_build_matrix(meta, existing_matrix=None, channel_sources=tuple()):
+    if meta.skip():
```
Comment (ocefpaf) [inline conda_smithy/configure_feedstock.py]: Seems sensible. Thanks for looking into this.

**Comment by johanneskoester** [comment]
Ok, I consider the issue to be fixed with this PR. If you agree, can we have a quick release? @pelson @jakirkham @ocefpaf.

**Comment by ocefpaf** [comment]
LGTM

**Comment by jakirkham** [comment]
Thanks @johanneskoester. A good fix indeed.

**Comment by johanneskoester** [comment]
Will there be a release, such that I can rely on it with upcoming R pull requests?

**Comment by jakirkham** [comment]
I wanted to sort out the other R issues. ( https://github.com/conda-forge/conda-smithy/issues/447 ) Otherwise we will just have different problems with R packages.

Also wanted to try and consolidate the team update code in `conda-smithy` ( https://github.com/conda-forge/conda-smithy/pull/466 ). So that we can setup something to manage team updates. Right now the current strategy is totally broken.

I don't know that I will have time to do all of that today. Maybe near week's end.

**Comment by johanneskoester** [comment]
Ok, that sounds great and makes a lot of sense. Thank you!

**Pull request merged**